homeassistant/components/pyscript/__init__.py

```diff
@@ -0,0 +1,293 @@
+"""Component to allow running Python scripts."""
+
+from collections import OrderedDict
+import glob
+import io
+import logging
+import os
+
+import voluptuous as vol
+import yaml
+
+from homeassistant.components.pyscript.eval import AstEval, EvalFunc
+from homeassistant.components.pyscript.event import Event
+from homeassistant.components.pyscript.handler import Handler
+from homeassistant.components.pyscript.state import State
+from homeassistant.components.pyscript.trigger import TrigInfo, TrigTime
+from homeassistant.const import (
+    EVENT_HOMEASSISTANT_STARTED,
+    EVENT_HOMEASSISTANT_STOP,
+    EVENT_STATE_CHANGED,
+    SERVICE_RELOAD,
+)
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.service import async_set_service_schema
+from homeassistant.loader import bind_hass
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = "pyscript"
+
+FOLDER = "pyscript"
+
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema(dict)}, extra=vol.ALLOW_EXTRA)
```

Comment:
No need to define a config schema if you're not validating anything.

Reply:
Thanks - removed in commit below.